common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -573,7 +573,7 @@ def available_translations(self, transcripts, verify_assets=True):
 
         # If we're not verifying the assets, we just trust our field values
         if not verify_assets:
-            translations = list(other_lang)
+            translations = list(other_lang) if other_lang else []
             if not translations or sub:
                 translations += ['en']
             return set(translations)
```

Comment:
Upon a bit of poking, and I am not a Python expert at all, it seems that when you initialize an empty collection `x` with [], then add elements to it, you still need to call `set(x)` to return it as a set, which I would assume is being done here to ensure that the data is not treated as reliably ordered. So given that we want to enforce that the return type is a set, _and_ given that we want to initialize the translations as an empty collection, we have to convert it to a set down below. The alternative is, I think, to initialize `translations` to a `set()` in the `else` case of this line. I defer to @jcdyer and/or @nedbat on that decision.